packages/apps/tools/src/components/Common/Layout/partials/Header/styles.css.ts

```diff
@@ -1,29 +1,29 @@
-import { darkThemeClass, sprinkles, vars } from '@kadena/react-ui/theme';
+import { atoms, darkThemeClass, tokens } from '@kadena/react-ui/theme';
 import { style } from '@vanilla-extract/css';
 
 export const walletConnectWrapperStyle = style([
-  sprinkles({
-    marginLeft: '$4',
+  atoms({
+    marginInlineStart: 'md',
   }),
 ]);
 
 export const headerButtonStyle = style([
-  sprinkles({
+  atoms({
     outline: 'none',
     background: 'none',
     border: 'none',
     cursor: 'pointer',
-    padding: '$3',
-    marginRight: '$2',
-    color: '$gray40',
+    marginInlineEnd: 'sm',
   }),
   {
+    color: tokens.kda.foundation.color.neutral.n50,
+    padding: tokens.kda.foundation.size.n3,
     selectors: {
       [`${darkThemeClass} &:hover`]: {
-        color: vars.colors.$white,
+        color: tokens.kda.foundation.color.neutral.n100,
       },
       [`&:hover`]: {
-        color: vars.colors.$white,
+        color: tokens.kda.foundation.color.neutral.n0,
```

Comment:
yeah, that's how it was before, but now they change depending on the theme. I see this is in the header so what you can do is apply the dark class and it will always stay in dark theme. If this needs to happen more often, we can try to find a different solution

Reply:
There already is such selector so that it always stays in dark theme

Reply:
if that's the case you shouldn't need to use a selector here to set both since the darkThemeClass will always be applied